### ISSUE TITLE
Failed to extract standard pose library when updating

### ISSUE THREAD

**Issue by MinaD153** [imchillin/Anamnesis#891]
I updated this program and it deleted the standard poses and my saved poses as well. I am not sure how to fix this. Is there a standard pose file I can input in to the Standard pose folder?

**To Reproduce**
Steps to reproduce the behaviour:

**Expected behaviour**

**Log file**
[2022-03-24-17-26-23.txt](https://github.com/imchillin/Anamnesis/files/8346047/2022-03-24-17-26-23.txt)

[2022-03-24-18-01-06.txt](https://github.com/imchillin/Anamnesis/files/8346038/2022-03-24-18-01-06.txt)

Drag and drop the related log file(s) into the window, and they will be uploaded as part of your bug report. Please avoid using screenshots, as they very rarely provide any information of use to the developers. If your issue would benefit from additional evidence, such as a pose or appearance file, please package them into a `.zip` archive and upload them as well.

**Comment by Ani-ki** [comment]
Can you please confirm *where* you were saving your own poses?

Standard poses are deleted and reextracted with every update.

**Comment by MinaD153** [comment]
AppData - Roaming- Anamnesis - Standard Poses

I had a folder in there and it deleted all of the poses. But the update did not include the basic standard poses. I receive an error that says "Failed to extract standard pose library"

**Comment by Ani-ki** [comment]
**At no point should you be placing pose files in the Built-In Pose folders. [This has been outlined specifically on the wiki.](https://github.com/imchillin/Anamnesis/wiki/Loading-and-Saving-Pose-Files#obtaining-pose-files)** Moving forward, please use the correct folder in `Documents/Anamnesis/Poses`, as this folder is generated for user-made and downloaded poses specifically, and is not touched when the tool is updated

Regarding the folder not being created on update: this is a known issue, and will be investigated.

**Comment by Yerdiss** [comment]
While the OP's error seems to be that the ver.txt file could not be found/opened, I had a similar error with a slightly different cause. But since I saw that all "_Failed to Extract Pose Library_" posts were closed as duplicates, I'll post this as comment here.

The error on my end was that the DateTime string inside of the **ver.txt** file could not be parsed. It gave a _"... was not recognized as a valid DateTime"_ error (in small print under the Failed to Extract Pose Library message). I'll paste the relevant log snippet below.

As a work-around, I edited the ver.txt file and replaced the DateTime pattern to match the one set on my OS. (In my case, _yyyy-MM-dd HH:mm:ss_). After restarting the tool, the Pose Library was extracted successfully.


`2022-04-30 12:29:47.518 +02:00 [ERR] Failed to extract standard pose library
System.FormatException: String '30/10/2021 9:44:57 am' was not recognized as a valid DateTime.
   at System.DateTimeParse.Parse(ReadOnlySpan'1 s, DateTimeFormatInfo dtfi, DateTimeStyles styles)
   at System.DateTime.Parse(String s)
   at Anamnesis.PoseModule.PoseService.ExtractStandardPoses()`
